### PR TITLE
Update the logic of checking whether vm is installed  and code refactoring

### PR DESCRIPTION
**Pull request by MadhukaHarith92**

### CODE REVIEW

File: components/cli/pkg/util/utils.go [hunk 1]
```diff
@@ -584,22 +584,19 @@ func ExecuteCommand(cmd *exec.Cmd, errorMessage string) error {
 	}()
 	err := cmd.Start()
 	if err != nil {
-		fmt.Printf("cellery : %v: %v \n", errorMessage, err)
-		os.Exit(1)
+		ExitWithErrorMessage("Error executing command", err)
```
Comment (anuruddhal) [inline components/cli/pkg/util/utils.go]: Shall we add the command name to the error message?

File: components/cli/pkg/util/utils.go [hunk 2]
```diff
@@ -584,22 +584,19 @@ func ExecuteCommand(cmd *exec.Cmd, errorMessage string) error {
...
 	}
 	return nil
 }
 
 func DownloadFromS3Bucket(bucket, item, path string, displayProgressBar bool) {
 	file, err := os.Create(filepath.Join(path, item))
 	if err != nil {
-		fmt.Printf("Error in downloading from file: %v \n", err)
-		os.Exit(1)
+		ExitWithErrorMessage("Failed to download from s3 bucket", fmt.Errorf("Error downloading from file", err))
```
Comment (anuruddhal) [inline components/cli/pkg/util/utils.go]: Shall we append to file name to the error message?

File: components/cli/pkg/util/utils.go [hunk 3]
```diff
@@ -904,14 +900,12 @@ func PrintSuccessMessage(message string) {
 func GetSourceFileName(filePath string) (string, error) {
 	d, err := os.Open(filePath)
 	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
+		ExitWithErrorMessage("Error opening file", err)
```
Comment (anuruddhal) [inline components/cli/pkg/util/utils.go]: Shall we add the filepath to the error message? Fix in other places as well

Comment (MadhukaHarith92) [reply components/cli/pkg/util/utils.go]: Updated

File: components/cli/pkg/util/utils.go (hunk 2 above)
Comment (MadhukaHarith92) [reply components/cli/pkg/util/utils.go]: Updated.

File: components/cli/pkg/util/utils.go (hunk 1 above)
Comment (MadhukaHarith92) [reply components/cli/pkg/util/utils.go]: Returned the error instead of exiting with error message.

**Pull request merged**